### DISCUSSION TITLE
Dúvida React: Header não está aparecendo

### DISCUSSION THREAD

**Discussion by evertonProgramadorCriativo** [frontendbr/forum#1580] [Perguntas e Respostas]
voce pode me ajuda a achar um error no meu projeto ,webpack
eu estou seguindo  a aula , com o professor porem na hora de aparecer o header nao apareceu  , eu ja comparei o codigo com o do professor   , precisso de  ajuda para identificar o por que  o header nao esta aparecendo no meu projeto
eu estou tentando descobrio o error a 3 dias
o codigo do professsor rodou no meu
https://github.com/evertonProgramadorCriativo/Galeria-de-fotos-bootstrap
esse e o meu

 
![Screenshot_4](https://user-images.githubusercontent.com/55323538/67634877-d0517a80-f89f-11e9-96c0-aab146b02ea2.png)

como demonstra na imagem
quando eu dou npm start
era para aparecer o header
esse e do professor

![Screenshot_5](https://user-images.githubusercontent.com/55323538/67634879-dc3d3c80-f89f-11e9-8838-0ad81761f13f.png)

aqui esta o codigo do professor
https://github.com/evertonProgramadorCriativo/professogaleria

GitHub
evertonProgramadorCriativo/professogaleria
Contribute to evertonProgramadorCriativo/professogaleria development by creating an account on GitHub.
eu ja comparei o codigo
e precisso de ajuda para identificar o error

**Comment by fdaciuk** [comment]
Oi @evertonProgramadorCriativo! Onde está o seu código?

**Comment by evertonProgramadorCriativo** [comment]
> https://github.com/evertonProgramadorCriativo/Galeria-de-fotos-bootstrap

**Comment by evertonProgramadorCriativo** [comment]
o de cima e meu codigo

**Comment by fdaciuk** [comment]
Esse link está dando 404 aqui.. não está privado esse repositório?

**Comment by evertonProgramadorCriativo** [comment]
https://github.com/evertonProgramadorCriativo/Galeria-de-fotos-bootstrap.git

**Comment by evertonProgramadorCriativo** [comment]
esta publico

**Comment by evertonProgramadorCriativo** [comment]
mandei outro link fdaciuk  https://github.com/evertonProgramadorCriativo/Galeria-de-fotos-bootstrap.git

**Comment by fdaciuk** [comment]
Tente acessar essa URL em uma aba anônima, que você vai ver que está dando 404 :)

**Comment by evertonProgramadorCriativo** [comment]
aba anonima 
--------------professor------------codigo-------------------Inicio----
https://github.com/evertonProgramadorCriativo/professogaleria
--------------professor------------codigo-------------------Fim----

---------------------------------------------------------------------------
--------------Meu------------codigo-------------------Inicio----

https://github.com/evertonProgramadorCriativo/bootstrapGaleria
--------------Meu------------codigo-------------------Fim----

**Comment by fdaciuk** [comment]
@evertonProgramadorCriativo seu arquivo `index.js` só está importando os estilos, nada mais. Não tem nenhum arquivo que tenha essa parte do header, você precisa escrever esse código :)

**Comment by evertonProgramadorCriativo** [comment]
por favor  
----------------esse ----e o meu codigo -------eu atualizei
 https://github.com/evertonProgramadorCriativo/meugaleria

**Comment by evertonProgramadorCriativo** [comment]
<html>

    <head>
        <meta charset="UTF-8">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        <meta http-equiv="X-UA-Compatible" content="ie=edge">
        <title>Document</title>
        <link rel="stylesheet" href="estilo.css" >
    </head>

    <body>
        <h1>dudu</h1>
        <header wm-include="/pages/template/header.html"></header>    
        
        <srcipt src="app.js"></srcipt>
    </body>

</html>

**Comment by evertonnppp** [comment]
@queroserninja  por favor  segue o link --->

https://github.com/evertonProgramadorCriativo/meugaleria

**Comment by evertonnppp** [comment]
Alguem saber resolver isso ?

**Comment by fdaciuk** [comment]
@evertonnppp o problema é nessa linha:
https://github.com/evertonProgramadorCriativo/meugaleria/blob/master/src/index.html#L16

Veja que você escreveu errado o que deveria ser `script` :)
